buildpack.md

```diff
@@ -901,6 +900,7 @@ command = "<command>"
 args = ["<arguments>"]
 direct = false
 default = false
+working-directory = "<working directory>"
```

Comment:
```suggestion
working-dir = "<working directory>"
```
adding this for consistency with @sclevine 's request on the platform spec PR https://github.com/buildpacks/spec/pull/290/files#r813238710